### ISSUE TITLE
Fuel Overdose Demo save bug

### ISSUE THREAD

**Issue by Triticum0** [RPCS3/rpcs3#13301]
### Quick summary

This game demo can't make it past the intro fmv's due to not having enough space available 

### Details

When testing this game I ran into an issue where the game wouldn't make it past the Intro due to not having enough available space, forcing the application to close. The space needed is: -41942051. This also affects the full game [NPEB00876] but updating the game to 1.05 fixes the issue. doing some research on youtube I found an original playthrough of the first demo which seems to be working fine so could only be an rpcs3 issue.

Rpcs3:
![image](https://user-images.githubusercontent.com/71127352/215164729-08f776b2-9a0b-4019-8363-e5bf706e1543.png)

PS3:
https://youtu.be/XpFHG6R1oRg

### Attach a log file

[RPCS3(2).log.gz](https://github.com/RPCS3/rpcs3/files/10521956/RPCS3.2.log.gz)

### System configuration

  SYS: RPCS3 v0.0.26-14568-1852b370 Alpha | master
    SYS: AMD Ryzen 5 2600 Six-Core Processor | 12 Threads | 15.95 GiB RAM | TSC: 3.400GHz | AVX+ | FMA3
    SYS: Operating system: POSIX, Name: Linux, Release: 5.15.85-1-MANJARO, Version: #1 SMP PREEMPT Wed Dec 21 21:15:06 UTC 2022
    SYS: Current Time: 2022-10-28T21:53:46
    RSX: Found vulkan-compatible GPU: 'NVIDIA GeForce RTX 3060 Ti' running on driver 526.47.0.0
    SYS: Firmware version: 4.89

### Other details

Note: I know the video is the first demo as the maps available in the first and second demos are different.

**Comment by AniLeo** [comment]
Can still reproduce

Could be related to savedata autoload, as these are the only fs operations happening before the error message thread is created

```
·E 0:00:36.529672 {PPU[0x1000004] Thread (S3DSavedataLoad) [HLE:0x0114492c, LR:0x0014f99c]} cellSaveData: cellSaveDataUserAutoLoad(version=0, userId=0, dirName=“NPEB90475SAVE_DATA”, errDialog=1, setBuf=*0xd0059f30, funcStat=*0xd9aab0, funcFile=*0xd9aaa8, container=0xffffffff, userdata=*0x0)
·! 0:00:36.529679 {PPU[0x1000004] Thread (S3DSavedataLoad) [HLE:0x0114492c, LR:0x0014f99c]} cellSaveData: savedata_op(): setBuf  = { .dirListMax=0, .fileListMax=16, .bufSize=896 }
·W 0:00:36.530584 {PPU[0x1000004] Thread (S3DSavedataLoad) [HLE:0x0114492c, LR:0x0014f99c]} cellSaveData: savedata_op(): funcStat returned result=1.
```

### System configuration

- RPCS3 v0.0.27-14983-d13c0ee3 Alpha | master
- AMD Ryzen 9 5900HX with Radeon Graphics | 16 Threads | 15.01 GiB RAM | TSC: 3.294GHz | AVX+ | FMA3
- Operating system: POSIX, Name: Linux, Release: 6.2.13-1-MANJARO

**Comment by ChwaraeTeg** [comment]
Still an issue as of RPCS3 v0.0.28-15395-bc40b61e Alpha: [FuelOverdoseDemo.log.gz](https://github.com/RPCS3/rpcs3/files/12164411/FuelOverdoseDemo.log.gz) Tried disabling the Native UI, also tried running with Interpreter (static) instead of Recompiler, issue still occurs.

Can confirm that on console the same version of the demo gets beyond this stage. After booting for the first time on console it created a (987 KB) folder at \dev_hdd0\home\0000000x\savedata\NPEB90475SAVE_DATA - this folder doesn't get created in RPCS3. Copying the folder from console to the corresponding RPCS3 folder allowed it to bypass the error and go in-game:

![Fuel Overdose Demo  NPEB90475](https://github.com/RPCS3/rpcs3/assets/104720038/dd1d4209-4e22-4bd7-b038-76a534ce90d4)
[FuelOverdoseDemoWithConsoleSave.log.gz](https://github.com/RPCS3/rpcs3/files/12164898/FuelOverdoseDemoWithConsoleSave.log.gz)

**Comment by ChwaraeTeg** [comment]
Fixed by #15313 it would appear

Log: [FuelOverdoseDemo.log](https://github.com/RPCS3/rpcs3/files/14628511/FuelOverdoseDemo.log)
